src/java.base/share/classes/java/util/zip/ZipEntry.java

```diff
@@ -37,6 +37,11 @@
 
 /**
  * This class is used to represent a ZIP file entry.
+ * <P>
+ * The combined length of the entry name, the extra field data, the
+ * entry comment and {@link ZipFile#CENHDR CEN Header size}, must not
```

Comment:
I think you flipped the usage of `@link` and `@linkplain` in this spec addition. `@link` renders code with hyperlink, and `@linkplain` renders like regular body font with hyperlink.

And the constant is available in this class too, so no need to link to `ZipFile` really.
```suggestion
 * entry comment and {@linkplain #CENHDR CEN Header size}, must not
```

Reply:
Good suggestion.  Updated, per your suggestion